techniques/general/G151.html

```diff
@@ -5,8 +5,8 @@
       <p>With this technique, a link to the transcript or script is provided and should conform to WCAG 2.0 and could either be included at another location on the same web page or at another URI.</p>
    </section><section id="examples"><h2>Examples</h2>
       <ul>
-         <li>A live radio play of a fringe theatre group is being broadcast to the Web. As the actors stick largely to a set script, and the budget for the program is small, the producers provide a link (with the playwright's permission) to the script of the play in HTML.</li>
-         <li>A member of the government broadcasts an important policy speech on the Web. A transcript of the speech is made available on the website when the speech starts.</li>
+         <li>A live radio play of a fringe theatre group is being broadcast to the web. As the actors stick largely to a set script, and the budget for the program is small, the producers provide a link (with the playwright's permission) to the script of the play in HTML.</li>
```

Comment:
"... to the web."

This feels like it verges on being more like proper noun usage? 
> lowercase except if proper noun meaning World Wide Web